### DISCUSSION TITLE
I disabled some weapons via the ufg inventory screen and i cant re-enable them. any advice? thanks in advance.

### DISCUSSION THREAD

**Discussion by Bagelman12** [Hydraxous/UltraFunGuns#38] [Q&A]
Idk how to re-enable them any help would be appreciated.

**Comment by Hydraxous** [comment]
1. What mod version are you using?
2. Are there any errors in the F8 console when you try to re-enable the weapons?
3. Are you using any other mods alongside UFG?

Some troubleshooting suggestions:
- Try to delete the loadout data file in the mod's directory.
- Try removing the mod entirely and re-installing it

**Reply by Bagelman12** [reply]
thx will try now

**Reply by Bagelman12** [reply]
i disabled some of the weapons using the ufg in game inventory and couldnt re-enable them if that helps.

**Reply by Hydraxous** [reply]
Thanks, it would also be helpful if you could provide the version number of the mod, it should show up in the games console (F8) when the game loads.

If you are using the latest version (1.2.0) you can attach the log.txt file from ULTRAKILL/BepInEx/plugins/UFG folder which would help greatly.
